docs/content/theme/styles/roboto.css

```diff
@@ -0,0 +1,26 @@
+
+body, input {
+    font-family: "Roboto",-apple-system,BlinkMacSystemFont,Helvetica,Arial,sans-serif;
+}
+
+code, kbd, pre {
+    font-family: "Roboto Mono",SFMono-Regular,Consolas,Menlo,monospace;
+}
+
+@font-face {
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Roboto'),
+         url('./Roboto/roboto-latin-regular.woff2') format('woff2'),
+         url('./Roboto/roboto-latin-regular.woff') format('woff'),
+}
+
+@font-face {
+    font-family: 'Roboto Mono';
+    font-style: normal;
+    font-weight: 400;
```

Comment:
Just as an idea, we could add `font-display: swap;` to reduce blocking while loading the fonts. Of course this would trigger a re-render but on really slow connections, might be a good option.

Reply:
Actually this is recommended by google lighthouse. Don't know why it isn't already included. I copy&pasted the font css.

Reply:
done